portfolio2023/index.css

```diff
@@ -0,0 +1,59 @@
+body {
+  background-color: rgb(178, 191, 153);
+}
+
+/* why do we include ul in our navbar class. Is it because its easier to 
+target ul like this instead of creating classes for the ul or for each 
+indiv li  */
+/* ul gives overall structure of the navbar */
```

Comment:
Some quick notes here,

- a `<navbar>` element is a semantic element that offers the explanation that the content inside this element is related to website navigation. However, you don't get that by using a `.navbar` class, you'd only get that for free if you used a `<navbar>` element instead of a class
- a `<ul>` defines semantic-wise that it is a list of directly related items; e.g. a list of pages that you can navigate to